### PR TITLE
mdbook migration

### PR DESCRIPTION
**Pull request by cor**
Migration from GitBook to mdBook with a custom theme. Since GitBook does not offer any export options, we had to copy & paste *all* content. Please review these contents carefully. Also includes a `flake.nix` for deploying the mdbook alongside the cargo docs. The flake also has a development shell for the main substrate development. There is also new content compared to the GitBook version, which describes how people can contribute to the new mdBook. 

Thanks @hussein-aitlahcen for working with me on the nix deployment and @PoisonPhang for helping with the (painful) content migration.

### CODE REVIEW

**Review by dzmitry-lahoda — APPROVED** [review]

**Comment by Dom-Roth** [comment]
It it possible to increase the space above headers a bit?
It's something i noticed on friday while working on the Angular docs, and just noticed again.
I think it'd make for better readability.

See Screenshots
Screenshot from 2022-03-25 20-15-54
gnome-shell-screenshot-h00b58

File: book/src/developer-guides.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# **💻** Developer Guides
+
+
+Here are the articles in this section:
+
+[Composable SDK](./developer-guides/composable-sdk.md)
```
Comment (Dom-Roth) [inline book/src/developer-guides.md:6]: Though this is correct with the live version, in this version there is another chapter called "Pallet Overviews".

**Comment by Dom-Roth** [comment]
Is it possible to configure all external links to be opened in a new tab, e.g. <a target="_blank">?

File: book/src/products/mosaic/fragmentation-in-layer-2.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Fragmentation in DeFi's Layer 2
+
+---
+
+While the Ethereum protocol’s revolutionary smart contract functionality is what makes DeFi possible, the ETH transaction speed for DApps is [notoriously slow](https://www.coindesk.com/learn/ethereum-101/will-ethereum-scale), and it stands in the way of further DeFi development and mass adoption. Just as a point of reference, Ethereum’s transaction throughput is about [1,500 times](https://hackernoon.com/the-blockchain-scalability-problem-the-race-for-visa-like-transaction-speed-5cce48f9d44) slower than that of Visa and about [250,000 times](https://www.barrons.com/articles/alibaba-records-25-3-billion-in-singles-day-sales-1510538618) slower than Alipay’s transaction throughput.
```
Comment (Dom-Roth) [inline book/src/products/mosaic/fragmentation-in-layer-2.md:5]: Though this is correct, the link `notriously slow` is linking a post that got pulled by CoinDesk in December.
Here's a link to the web archive for it:
https://web.archive.org/web/20210816233523/https://www.coindesk.com/learn/ethereum-101/will-ethereum-scale

File: book/src/products/mosaic/proof-of-concept-polygon-arbitrum-cross-layer-transferral-system.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
+# Proof of Concept: A Cross-Layer Transferal System
+
+---
+
+We have successfully initiated our Layer 2 - Layer 2 asset transferal system (Mosaic) with a Proof of Concept (PoC) linking Polygon, Fantom, Arbitrum, the Avalanche C-Chain, Moonriver, and the Ethereum mainnet (layer 1). 
+
+Polygon, Arbitrum, and Avalanche are leading L2/scalability solutions for Ethereum, with all being used by many of the largest DeFi protocols. For instance, [nearly 100 protocols](https://www.block123.com/en/feature/polygon-matic-network-list/) have built on Polygon including MakerDAO's [DAI](https://makerdao.com/en/) (with [MakerDAO](https://makerdao.com/)being the second largest DeFi protocol at the time of writing, having over $13.5 billion in TVL, or total value locked), [Chainlink](https://chain.link/), and [Aavegotchi](https://aavegotchi.com/). Arbitrum has been incorporated into major DeFi protocols like [Bancor](https://bancor.network/) (currently the 13th largest DeFi project in terms of TVL), [Augur](https://augur.net/) and others. Avalanche similarly has an impressive [343 protocols](https://www.avax-projects.com/) built along it at the time of writing.
```
Comment (Dom-Roth) [inline book/src/products/mosaic/proof-of-concept-polygon-arbitrum-cross-layer-transferral-system.md:7]: The `nearly 100 protocols` link doesn't DNS resolve for me, can someone check?

File: book/src/products/mosaic/fragmentation-in-layer-2.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Fragmentation in DeFi's Layer 2
+
+---
+
+While the Ethereum protocol’s revolutionary smart contract functionality is what makes DeFi possible, the ETH transaction speed for DApps is [notoriously slow](https://www.coindesk.com/learn/ethereum-101/will-ethereum-scale), and it stands in the way of further DeFi development and mass adoption. Just as a point of reference, Ethereum’s transaction throughput is about [1,500 times](https://hackernoon.com/the-blockchain-scalability-problem-the-race-for-visa-like-transaction-speed-5cce48f9d44) slower than that of Visa and about [250,000 times](https://www.barrons.com/articles/alibaba-records-25-3-billion-in-singles-day-sales-1510538618) slower than Alipay’s transaction throughput. 
+
+Several blockchain scaling solutions have been proposed to solve this scalability problem, including [State Channels](https://l4.ventures/papers/statechannels.pdf), [Sharding](https://www.bubifans.com/ueditor/php/upload/file/20181015/1539597837236127.pdf), [Plasma](https://plasma.io/plasma.pdf), [OVM](https://optimism.io/), and [ZKRollups](https://github.com/PlasmNetwork/ZKRollups). These scaling solutions can be categorized as Layer 2 scaling solutions.
```
Comment (Dom-Roth) [inline book/src/products/mosaic/fragmentation-in-layer-2.md:7]: The link `Sharding` loads infinitely for me, can someone check?

**Review by Dom-Roth — CHANGES_REQUESTED** [review]

File: book/src/developer-guides.md (hunk 1 above)
Comment (PoisonPhang) [reply book/src/developer-guides.md:6]: These are a recent addition from me that I had finished after the migration began. Part of a larger goal making our technical documentation more approachable. Would we prefer that this be moved to its own PR?

File: book/src/ecosystem/the-composable-team.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+### Karel Kubat | CTO 
+
+Karel is a long-term crypto user and developer, passionate about decentralization through technological means. Karel has worked on a range of protocols and contracts, such as Ethereum, NEO, Substrate, and custom chains. He brings this expertise to the Composable team, working to help democratize financial instruments and markets.
+
+### Zain | CMO 
+
+Zain is a seasoned art, brand and marketing director who has worked between the intersection of tech and development. Previously head of brand and communications at Biconomy, marketing head at Muzmatch (Y Culimantor), Zain has built award-winning campaigns and marketing strategies for his innovative organizations and businesses. He brings a wealth of industry and related experience to the role.
```
Comment (benluelo) [inline book/src/ecosystem/the-composable-team.md:21]: Y Combinator?

File: book/src/audits/code-fixes.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+
+![type MinimumPeriod](./type-minimum-period.png)
+
+The reason it was 6 seconds in the first place was that our runtime was first a standalone substrate blockchain and had a 6 seconds expected block time. Unfortunately, when it was converted to a parachain, this value wasn’t updated to Parachains standard block time. Therefore, this caused spotty block production on Picasso, with block times averaging 30 seconds. When viewed even more critically, this makes sense since the collators are authoring blocks every 6 seconds but what is happening behind the scenes is that they are really only allowed to produce blocks every 12 seconds by the relay chain, so collators often miss their slots.
+
+After the runtime upgrade, all our collators stopped producing blocks. This was primarily because, from the perspective of the parachain, the new slot numbers produced by the new runtime code are for earlier slots.
+
+From the logs of one of the collators: 2021–12–18 05:54:00.506 DEBUG tokio-runtime-worker aura: [Parachain] Starting authorship at slot 136650570; timestamp = 1639806840450
```
Comment (benluelo) [inline book/src/audits/code-fixes.md:26]: can probably format the log line better

File: book/src/audits/code-fixes.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+
+![aura.currentSlot](./aura-current-slot.png)
+
+It would take roughly 53 years for our chain to start producing blocks.
+
+With the chain unable to produce blocks, it became clear we had only two options:
+
+**1. Reset our Parachain runtime WASM on the relay chain with a WASM file that reverts the changes to the MinimumPeriod. Then, create a new chain-spec with that same WASM blob specified as a WASM override, then update our collators so they can resume producing blocks.**
+
+**Pros**
```
Comment (benluelo) [inline book/src/audits/code-fixes.md:38]: probably want a newline before the list items

File: book/src/faqs/faqs.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+
+Leading this round were [Advanced Blockchain AG](https://en.advancedblockchain.com/) and [Rarestone Capital](https://rarestone.capital/), both of which are key venture capitalists investing in the blockchain industry. Together these organizations used strategic investors to make this funding round a success.
+
+### How was Composable Finance created?
+
+Through its subsidiaries, [Advanced Blockchain AG](https://en.advancedblockchain.com/) incubated Composable Finance on the technical side and provided the intellectual property for Composable. During this period, Cosmin Grigore was the Lead Developer on the project. This incubation was completed alongside top-tier Web 3.0-focused venture capital firm [Rarestone Capital](https://rarestone.capital/), which assisted with product strategy and rollout for Composable Finance. Subsequently, Composable Finance was spun out, retaining Cosmin as its CEO. Advanced Blockchain also reinvested into Composable. 
+
+Are there any socials I can follow for any additional questions and get more updates and information?
```
Comment (benluelo) [inline book/src/faqs/faqs.md:36]: should this be a header or something? maybe a quote (`>`)

**Review by benluelo — COMMENTED** [review]
Not sure if having the images in-repo is a great idea, it's more for people to download when cloning the repo. Other than that, looks good to me :+1:

File: book/src/developer-guides.md (hunk 1 above)
Comment (Dom-Roth) [reply book/src/developer-guides.md:6]: Completely fine with adding this to the PR. Sorry been a bit unclear. Just wanted to have the link added to this index doc.

I don't know how to suggest changes on GitHub, so this is basically what i intended to have here:
```
- [Composable SDK](./developer-guides/composable-sdk.md)
- [Pallet Overviews](./developer-guides/pallet-overviews.md)
```

File: book/src/products/mosaic/proof-of-concept-polygon-arbitrum-cross-layer-transferral-system.md (hunk 3 above)
Comment (Dom-Roth) [reply book/src/products/mosaic/proof-of-concept-polygon-arbitrum-cross-layer-transferral-system.md:7]: There is an old version in the web archive. But it looks like this was an ever evolving page, showing different Polygon projects. So I'm not sure if it'd make sense, to replace this link with the archive one.
https://web.archive.org/web/20210725112237/https://www.block123.com/en/feature/polygon-matic-network-list/

**Comment by cor** [comment]
> Is it possible to configure all external links to be opened in a new tab, e.g. `<a target="_blank">`?

I'm not sure about this one. Is this behavior we want, when users can also choose to manually open links in a new tab? (cmd click). Also, when we start crosslinking between the rustdocs and the mdbook, do we consider the rustdocs to be "external"? are medium posts by 0xbrainjar "external"?

File: book/src/products/mosaic/fragmentation-in-layer-2.md (hunk 4 above)
Comment (cor) [reply book/src/products/mosaic/fragmentation-in-layer-2.md:7]: Not sure what to do about this one. this is the link that was in the original doc. Perhaps I should just remove the link?

File: .vscode/tasks.json [hunk 9]
```diff
@@ -0,0 +1,14 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Serve Book",
+            "type": "shell",
+            "command": "nix develop \".#book\" --command mdbook serve ./book --open",
```
[the remark this answers is not in the document]
Comment (cor) [reply .vscode/tasks.json:7]: You should run it from the project root in order to get it to work, hence the `./book` argument :)

File: book/src/developer-guides.md (hunk 1 above)
Comment (cor) [reply book/src/developer-guides.md:6]: I like this suggestion, will add it @Dom-Roth

Comment (cor) [reply book/src/developer-guides.md]: Fixed in 2fb2435b24b17e33f77f2f9e7f5782bdb694f442

File: book/src/products/mosaic/fragmentation-in-layer-2.md (hunk 2 above)
Comment (cor) [reply book/src/products/mosaic/fragmentation-in-layer-2.md:5]: removed this link

File: book/src/products/mosaic/proof-of-concept-polygon-arbitrum-cross-layer-transferral-system.md (hunk 3 above)
Comment (cor) [reply book/src/products/mosaic/proof-of-concept-polygon-arbitrum-cross-layer-transferral-system.md:7]: replaced link with archive version

File: book/src/products/mosaic/fragmentation-in-layer-2.md (hunk 4 above)
Comment (cor) [reply book/src/products/mosaic/fragmentation-in-layer-2.md:7]: I've decided to remove the link

File: book/src/SUMMARY.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+- [Cross-Chain Virtual Machine](./products/cross-chain-virtual-machine.md)
+    - [Innovation Availability Layer](./products/cross-chain-virtual-machine/innovation-availability-layer.md)
+- [Routing Layer](./products/routing-layer.md)
+- [Mosaic](./products/mosaic.md)
+    - [Fragmentation in DeFi's Layer 2](./products/mosaic/fragmentation-in-layer-2.md)
+    - [Proof of Concept: A Cross-Layer Transferal System](./products/mosaic/proof-of-concept-polygon-arbitrum-cross-layer-transferral-system.md)
+        - [TriCrypto, aToken, and SLP PoC Expansions](./products/mosaic/proof-of-concept-polygon-arbitrum-cross-layer-transferral-system/curve-tricrypto-poc-expansion.md)
+        - [PoC Components and User Flow](./products/mosaic/proof-of-concept-polygon-arbitrum-cross-layer-transferral-system/poc-components-and-user-flow.md)
+        - [Poc Walkthrough](./products/mosaic/proof-of-concept-polygon-arbitrum-cross-layer-transferral-system/poc-walkthrough.md)
+        - [PoC vault and liquidity rebalancing process](./products/mosaic/proof-of-concept-polygon-arbitrum-cross-layer-transferral-system/poc-vault-and-liquidity-rebalancing-process.md)
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/SUMMARY.md:22]: Done, thank you

File: book/src/products/cross-chain-virtual-machine/innovation-availability-layer.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
...
+Components include:
+
+- IBC-Substrate pallets
+- Beefy-IBC client merged into IBC-Go
+- Beefy-Go light clients
+- Mint and burn approach for transfers
+
+
+![IBC-Substrate Bridge](./ibc-substrate-bridge.png)
+*The IBC-Substrate Bridge. Note that the relayer is just a process to pass messages, and does not perform signing, etc*
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/cross-chain-virtual-machine/innovation-availability-layer.md:31]: done, thanks

File: book/src/products/the-picasso-parachain.md [hunk 12]
```diff
@@ -0,0 +1,125 @@
...
+pre-built components. Think of pallets as stackable building blocks that any 
+developer can use to easily build protocols and dApps.
+
+Although smart contracts continue to serve vital and critical functions through 
+their deterministic and built-in parts of blockchains’ consensus mechanisms, 
+they add an additional layer of logic on top of an existing blockchain’s core 
+logic, which creates some inflexibility. While smart contracts do an excellent 
+job at triggering specific trustless agreements and subsequent automated 
+actions, developers prefer greater flexibility — which pallets provide. 
+Moreover, palettes provide a more robust and powerful framework to help propel
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/the-picasso-parachain.md:53]: done

File: book/src/products/parachain-vault-strategy/parachain-auction-tokens.md [hunk 13]
```diff
@@ -0,0 +1,31 @@
+# Parachain Auction Tokens
+
+**For transparency, we are continuing to publicize the parachain vault approach 
+we took when bidding for a Kusama parachain. This strategy is no longer being 
+implemented to support Kusama parachian procurement, as this milestone has
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/parachain-vault-strategy/parachain-auction-tokens.md:5]: fixed, thanks

File: book/src/products/parachain-vault-strategy/walkthroughs.md [hunk 14]
```diff
@@ -0,0 +1,171 @@
...
+**For transparency, we are continuing to publicize the parachain vault approach 
+we took when bidding for a Kusama parachain. This strategy is no longer being 
+implemented to support Kusama parachian procurement, as this milestone has 
+already been achieved. Please note that the information in italicized text below 
+is outdated for this reason:**
+
+## *General Platform/Site Access*
+
+*To access the Composable Finance platform and its features, visit its webpage 
+at https://www.composable.finance/:*
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/parachain-vault-strategy/walkthroughs.md:12]: fixed

File: book/src/products/parachain-vault-strategy/risks.md [hunk 15]
```diff
@@ -0,0 +1,60 @@
...
+KSM/DOT auctions, opening gates that were chained shut in the past.*
+
+*Opportunities for yield farming have inherent risks associated with them. 
+Please take note of the following risks associated with our solution:*
+
+## *Smart Contract Related Risks*
+
+*Even though smart contracts are known for being a stable and a secure way of 
+processing data, bugs can still be discovered.*
+*Attempted remediation for this risk includes the following:*
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/parachain-vault-strategy/risks.md:20]: Good catch! fixed

File: book/src/products/the-picasso-parachain/the-picasso-crowdloan/crowdloan-how-to-ksm.md [hunk 16]
```diff
@@ -0,0 +1,61 @@
...
+the information in italicized text below is outdated for this reason:**
+
+*This page outlines the simple process for participating in the [Picasso 
+Crowdloan](../the-picasso-crowdloan.md) using KSM. The following link is a 
+how-to-guide that guides users through the process of participating in our 
+crowdloan:*
+
+*https://www.youtube.com/watch?v=DG2GLYVdT4U*
+
+*Our crowdloan is available at https://crowdloan.composable.finance/. Click on
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/the-picasso-parachain/the-picasso-crowdloan/crowdloan-how-to-ksm.md:15]: Thank you, fixed

File: book/src/products/the-picasso-parachain/the-picasso-crowdloan/crowdloan-how-to-stablecoins.md [hunk 17]
```diff
@@ -0,0 +1,130 @@
...
+parachain auction. Users will receive their funds back, in addition to PICA 
+rewards should we win the auction. These processes are fully detailed below.*
+
+## How To Guide: Participating in the Picasso Crowdloan with ERC-20 Stablecoins
+
+*Once our ERC-20 stablecoin vault goes live, users will be able to contribute 
+USDC, USDT, and DAI. The cap will be $15 million for the first week. Afterwards, 
+we will raise the cap to $50 million.*
+
+*Our vault will be available at 0x4adA5227e164a37a0183eC6Ce93222232f6A00F1. To
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/the-picasso-parachain/the-picasso-crowdloan/crowdloan-how-to-stablecoins.md:34]: Thank you, I've made them monospace

File: book/src/products/the-picasso-parachain/the-picasso-crowdloan/crowdloan-how-to-stablecoins.md [hunk 18]
```diff
@@ -0,0 +1,130 @@
...
+these purchases of KSM are executed, as well as the price ranges in which these 
+purchases were made. This KSM will then be contributed directly to the parachain 
+slot for Composable’s Picasso parachain.*
+
+*[Three Arrows Capital](https://www.threearrowscap.com/) will be the capital 
+provider for making these KSM purchases. This nearly decade-old hedge fund 
+manager focuses on investments into the blockchain space, particularly in crypto 
+and DeFi. The Composable and Three Arrows Capital teams will be on a multisig 
+wallet on Ethereum at 0xAD44c5bb2c726Fcecf67D66Baa6cD8cC447A1637 and on Kusama at
+FxLmDZbMB7sZfR6MJbuUzRmrb535nKcT2CxyGwWF6LLieud. The purchase log can be found
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/the-picasso-parachain/the-picasso-crowdloan/crowdloan-how-to-stablecoins.md:105]: Thank you, I've made them monospace

File: book/src/products/the-composable-parachain/the-composable-crowdloan.md [hunk 19]
```diff
@@ -0,0 +1,66 @@
+# The Composable Crowdloan
+
+Our crowdloan on Polkadot was critical in allowing us to successfully obtain the 
+8th Polkadot parachain slot during the 7th round of auctions of these 
+parachains. Over 9 thousand users participated in our crowdloan, depositing a 
+total of 6,075,485 DOT (a value of over $163 million).
+
+**For transparency, we are continuing to publicize the crowdloan approach we took 
+when bidding for a Polkadot parachain. Please note that the information in 
+italicized text below is outdated for this reason**:
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/the-composable-parachain/the-composable-crowdloan.md:10]: done, thanks

File: book/src/products/the-composable-parachain/the-composable-crowdloan.md [hunk 20]
```diff
@@ -0,0 +1,66 @@
...
+
+*This crowdloan was available on our [app](https://crowdloan.composable.finance/). 
+Users were able to stake stablecoins or DOT, earn an APY on their stakings, and 
+support the ever-growing Composable ecosystem. Our Polkadot crowdloan went live 
+within a few days of its announcement, allowing users to bid in the second 
+batch.*
+
+## Rewards
+
+***We initially offered offering 12% of LAYR supply to crowdloan contributors,
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/the-composable-parachain/the-composable-crowdloan.md:27]: if done some fixed fixing :)

File: book/src/products/the-composable-parachain/the-composable-crowdloan.md [hunk 21]
```diff
@@ -0,0 +1,66 @@
...
+
+***We decided that, if we passed the 4 million DOT staked mark, we would add an 
+additional 2% of total LAYR supply to crowdloan contributors. We ended up 
+significantly surpassing this milestone, thus unlocking these additional 
+rewards.***
+
+***With the extra 2% increase thus unlocked, it means we increased our total 
+LAYR supply distributed to crowdloan supporters from 12% to 14%.***
+
+*As [stated in a previous article](https://composablefi.medium.com/composable-enables-participation-in-the-picasso-crowdloan-using-erc-20-stablecoins-launching-aab49c965a1a#:~:text=PICA%20rewards%20will%20be%20distributed%20to%20Polkadot.js%20wallets%20through%20a%20claim%20process.),
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/the-composable-parachain/the-composable-crowdloan.md:60]: I'm not sure honestly, but the link works. Maybe it is for SEO? Let's just leave it in there for now

File: book/src/products/the-composable-parachain/the-composable-crowdloan/crowdloan-how-to-guide-for-stablecoin.md [hunk 22]
```diff
@@ -0,0 +1,86 @@
...
+*Our crowdloan campaign to secure a Polkadot parachain mirrored the strategy of 
+our previously successful procurement of our Kusama parachain, Picasso. As with 
+Picasso, our Composable Parachain crowdloan was the first project to accept 
+stablecoins as crowdloan contributions for the parachain auctions on Polkadot, 
+thus opening up the Dotsama ecosystem to a broader audience.*
+
+*Below is a walk-through on how to participate using the ERC-20 stablecoins: 
+**USDT**, **DAI** and **USDC**.*
+
+## Enter into the Polkadot Ecosystem
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/the-composable-parachain/the-composable-crowdloan/crowdloan-how-to-guide-for-stablecoin.md:16]: done

File: book/src/products/the-composable-parachain/the-composable-crowdloan/crowdloan-how-to-guide-for-stablecoin.md [hunk 23]
```diff
@@ -0,0 +1,86 @@
...
+  (until Friday 17th Dec 21:00 EST)*
+
+* *And 20% referral bonus to users who refer a sum of 1000 DOT using our 
+  referral codes*
+
+*You can find more details about our Polkadot [parachain procurement plans and 
+LAYR token distribution](https://composablefi.medium.com/announcing-the-composable-polkadot-parachain-procurement-plans-a4e8dd82c4fe) 
+in case you missed it.*
+
+## Be a part of Composable’s Polkadot parachain journey
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/the-composable-parachain/the-composable-crowdloan/crowdloan-how-to-guide-for-stablecoin.md:46]: thansk, done

File: book/src/products/the-composable-parachain/the-composable-crowdloan.md [hunk 24]
```diff
@@ -0,0 +1,66 @@
...
+for participants. For our Polkadot crowdloan, participation was capped at 25 
+million DOT.*
+
+*This crowdloan was available on our [app](https://crowdloan.composable.finance/). 
+Users were able to stake stablecoins or DOT, earn an APY on their stakings, and 
+support the ever-growing Composable ecosystem. Our Polkadot crowdloan went live 
+within a few days of its announcement, allowing users to bid in the second 
+batch.*
+
+## Rewards
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/products/the-composable-parachain/the-composable-crowdloan.md:25]: done

File: book/src/governance-and-tokenomics/the-picasso-pica-network-token.md [hunk 25]
```diff
@@ -0,0 +1,104 @@
...
+
+Collators that create blocks on the parachain, as well as Picasso holders who stake Picasso with collators, will earn in protocol rewards. More specifically, Collators will earn transaction fees - a portion of which will be burnt, and also flow into the treasury.
+
+Additionally, similar to inflation in Polkadot, there will be an inflation curve that has exponential rewards. Oracle providers in our system will earn rewards from inflation.
+
+***The team will receive 15% of supply (1,500,000,000 tokens) as well.***
+
+***The treasury, which will be used to incentivize the initial 5-6 projects that deploy on the Picasso network, along with pallet creation grants, will be allocated 15% (1,500,000,000 tokens)***. Furthermore, as mentioned previously, a certain percentage of transaction fees will go towards the treasury.
+
+## Vesting Distribution
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/governance-and-tokenomics/the-picasso-pica-network-token.md:57]: done

File: book/src/governance-and-tokenomics/layr-tokens.md [hunk 26]
```diff
@@ -0,0 +1,57 @@
+# LAYR Tokens
+
+*LAYR is Composable's Polkadot and ETH Token.*
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/governance-and-tokenomics/layr-tokens.md:3]: this is intentional

File: book/src/governance-and-tokenomics/layr-tokens.md [hunk 27]
```diff
@@ -0,0 +1,57 @@
...
+
+### Protocol Emissions
+
+10% (or 10,000,000 LAYR) of the total token supply will be released from the protocol as rewards and incentives for a number of actions involved in the protocol. These are programmatic incentives to bootstrap network growth (block validators, decentralized application builders and token holders from other networks) on Composable as well as token liquidity (liquidity mining programs). These tokens will be distributed evenly each month for 36 months (approximately 277,777.778 LAYR per month for three years).
+
+### Crowdloans
+
+15% (or 12,000,000 LAYR) of the total token supply will be allocated for the purposes of securing a parachain slot every two years. 12% is being utilized for the current batches of Polkadot auctions, with a 25% vesting on TGE, and the remainder vesting over two years.
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/governance-and-tokenomics/layr-tokens.md:45]: was a mistake in the original docs, i've fixed it

File: book/src/roadmap/timeline.md [hunk 28]
```diff
@@ -0,0 +1,9 @@
+# Timeline
+
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/roadmap/timeline.md:2]: done

File: book/src/roadmap/timeline.md [hunk 29]
```diff
@@ -0,0 +1,9 @@
...
+The next phase for Composable is to integrate Picasso with the [inter-blockchain communication protocol](https://ibcprotocol.org/) (IBC). Through this step, we will enable cross-chain communication with every blockchain that uses IBC.
+
+To do this, we will be utilizing Substrate to create the bridge infrastructure to connect our parachain to IBC. Beyond this, we will work to improve Mosaic’s cross-layer communication so that our transferal system can connect with our IBC-Picasso bridge. With this integration, we will look to generalize Mosaic to be a liquidity model that will extend across ecosystems.
+
+From there, we will begin drawing on the ink! VM model to develop the Composable Cross-Chain VM, which will spring forth the ability f
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/roadmap/timeline.md:11]: yes its unfinished, but let's keep it like that for now. purpose of this PR is the migration

File: book/src/ecosystem/build-on-composable-ecosystem-development.md [hunk 30]
```diff
@@ -0,0 +1,95 @@
...
+
+Composable Finance has built an [SDK](../developer-guides/composable-sdk.html) with Mosaic in order to facilitate development on layer 2 (L2). Presently, Mosaic is conducting its [proof of concept](https://composablefi.medium.com/testing-mosaic-through-our-arbitrum-polygon-transferal-system-proof-of-concept-4e459e24e427) (PoC) and on its way to full operability.
+
+With Mosaic, we are enabling assets and information to move seamlessly across L2 solutions. As a result, movement between all of the many decentralized exchanges, liquidity pools, and other DeFi protocols is possibl.
+
+This dispatch system will also eventually enable cross-layer function calls. This is crucial infrastructure which allows for individual smart contracts from disparate layer 2 solutions to directly communicate. Smart contracts on [Polygon](https://polygon.technology/), for example, will be able to talk to smart contracts on [Arbitrum](https://offchainlabs.com/).
+
+Given this wide scope of possibilities, our cross-layer grant recipients can have a diverse set of interests:
+
+*Yield farming enthusiast*? Use Composable’s SDK to build a cross-layer yield aggregator, allowing for maximal yield generation.
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/ecosystem/build-on-composable-ecosystem-development.md:76]: not meant to be in italics

File: book/src/ecosystem/build-on-composable-ecosystem-development.md [hunk 31]
```diff
@@ -0,0 +1,95 @@
...
+
+*Crypto-lending connoisseur*? Use cross-layer function calls to build a flash loan protocol which can leap across L2s within a single block time.
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/ecosystem/build-on-composable-ecosystem-development.md:78]: not meant to be in italics

File: book/src/ecosystem/the-composable-team.md (hunk 5 above)
Comment (cor) [reply book/src/ecosystem/the-composable-team.md:21]: hmm it's also Y Culimantor on the original page

Comment (cor) [reply book/src/ecosystem/the-composable-team.md:21]: changed to Y Combinator

File: book/src/faqs/faqs.md (hunk 8 above)
Comment (cor) [reply book/src/faqs/faqs.md:36]: done

File: book/src/ecosystem/business-line-development.md [hunk 32]
```diff
@@ -0,0 +1,35 @@
...
+
+---
+
+Composable makes it easy for other innovative projects to join our ecosystem and build with our technology. This allows other developers in the space to make their own applications interoperable and more intuitive, vastly improving the cross- ecosystem user experience.
+
+Composable offers the following services for other businesses, projects, and community developers:
+
+### Data provider onboarding:
+
+Onboarding individuals/entities that can provide data for our oracle pallet, [Apollo](../products/the-picasso-parachain/the-picasso-tech-stack/apollo.html), like Teller and API3.
```
[the remark this answers is not in the document]
Comment (cor) [reply book/src/ecosystem/business-line-development.md:11]: it works

**Review by benluelo — APPROVED** [review]

**Review by PoisonPhang — APPROVED** [review]

**Pull request merged**